ui/src/childinfo/ChildInfo.js

```diff
@@ -19,27 +18,24 @@ export default class ChildInfo extends Component {
         rocketContainerHeight: 350,
         isBlastOff: false
       }
+      this.numSteps = Object.keys(this.stepMapFunction()).length
     }
 
     stepMapFunction = () => {
       return {
         0: {
-          text: 'Hi what is your name?',
-          input: 'name'
+          text: `Hi${this.props.name ? ` ${this.props.name}` : ''}, I have a few questions for you before we can make your wish come true!`,
```

Comment:
can we destructure name prop to be consistent?

Reply:
Sure Ill do that